src/ui/hexeditwidget.cc

```diff
@@ -302,6 +319,13 @@ void HexEditWidget::initParsersMenu() {
   }
 }
 
+void HexEditWidget::initUnprintablesMenu() {
+  unprintables_menu_.clear();
+  unprintables_menu_.addAction("dots");
+  unprintables_menu_.addSeparator();
```

Comment:
Why a separator here? We have only two options, so it not too useful now ;)

Reply:
Removed.